cura/Settings/MachineManager.py

```diff
@@ -1368,15 +1368,12 @@ def applyRemoteConfiguration(self, configuration: ConfigurationModel) -> None:
                                                                                        configuration.buildplateConfiguration,
                                                                                        extruder_configuration.material.guid)
 
-                if variant_container_node:
+                # If the remote machine has an empty variant, do nothing
+                if variant_container_node is not None:
                     self._setVariantNode(position, variant_container_node)
-                else:
-                    self._global_container_stack.extruders[position].variant = empty_variant_container
```

Comment:
Uh, but if `variant_container_node` was None, the variant is retained from whatever it was previously?